This patch corrects the scripts for versioning of the JAR file name

--- scripts/run-antlr.in-orig	Wed Nov  1 15:37:18 2006
+++ scripts/run-antlr.in	Tue Feb  8 19:13:27 2022
@@ -49,7 +49,7 @@
 PACKAGE_VERSION="@PACKAGE_VERSION@"
 versioneddir="@PACKAGE_NAME@-@PACKAGE_VERSION@"
 
-ANTLR_JAR="@libdir@/antlr.jar"
+ANTLR_JAR="@libdir@/antlr-@PACKAGE_VERSION@.jar"
 
 case @build_os@ in
   cygwin|mingw|msys)
